src/config/seo/index.ts

```diff
@@ -1,7 +1,7 @@
 import { DefaultSeoProps } from 'next-seo'
 const title = 'Eden Smart Proxy'
```

Comment:
would these values be better to set as env vars?

Reply:
Normally we use them burned but we can pass them to environment variables, it seems better to me.